ext/soup/gstsouphttpclientsink.c

```diff
@@ -218,7 +218,7 @@ gst_soup_http_client_sink_init (GstSoupHttpClientSink * souphttpsink,
   souphttpsink->proxy_id = NULL;
   souphttpsink->proxy_pw = NULL;
   souphttpsink->prop_session = NULL;
-  souphttpsink->timeout = 1;
+  souphttpsink->timeout = 30;
```

Comment:
Here you are not using a define as in souphttpsrc

Reply:
But in this case I only changed the value, not added the assignment. So it is exactly like it was before I changed the value 1 for 30.

Reply:
ok